### ISSUE TITLE
Error with oh-my-zsh

### ISSUE THREAD

**Issue by aarenhofferth** [brock/node-reinstall#15]
I've tried to run node-reinstall, and everything appears to run well until it gets to sourcing, where I receive this error:
```
Sourcing your ~/.zshrc file.
/Users/aaren/.oh-my-zsh/oh-my-zsh.sh: line 26: syntax error near unexpected token `('
/Users/aaren/.oh-my-zsh/oh-my-zsh.sh: line 26: `for config_file ($ZSH/lib/*.zsh); do'
```
After this error is received, I also noticed my node-reinstall file is removed from my /usr/local/bin.

Any ideas? Thanks!

**Comment by brock** [comment]
@aarenhofferth 

Regarding the `node-reinstall` location:
The installer copies the actual `node-reinstall` to `$HOME/bin/node-reinstall`. Line 114 of the script removes anything in `$PREFIX/bin/node*`, so if it ever was installed there, that's why it is no longer there.

Regarding the oh-my-zsh error:
I use oh-my-zsh as well, and I've not seen that error. Looking at the [source code in oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh/blob/master/oh-my-zsh.sh#L26), I'd guess it could be related to one of two things:
  1. Either the `$ZSH` variable has gotten unset (`node-reinstall` doesn't do that though)
  2. There might possibly be a script found when running `$ZSH/lib/*.zsh` that doesn't "validate" (for lack of a better word).

You should be able to confirm that you have that directory and files in it: `cd $ZSH/lib`

**Comment by aarenhofferth** [comment]
Ok great I'll check that out. Thanks!

**Comment by brock** [comment]
@aarenhofferth what OS are you running this on?

**Comment by aarenhofferth** [comment]
Mac OSX 10.10.3 (Yosemite)

**Comment by brock** [comment]
I doubt it is related, @aarenhofferth, but I pushed a fix that prevents the installer from deleting itself from `/usr/local/bin`. Let me know if you still have issues.

**Comment by brock** [comment]
@aarenhofferth I tested this again, and when I scrolled back up through the logs, I found that I too was receiving that error. After looking at this, I'm removing the "sourcing" of the profile logic because I'm 99% certain that it doesn't have any effect anyway. Sourcing the script happens inside the sub-shell that runs when node-reinstall is run. But as soon as it returns us back to the primary shell, those sourced values aren't available anyway.

Version 0.0.7 fixes the error. Thanks!
